.gitignore

```diff
@@ -2,6 +2,9 @@
 !.gitignore
 !.travis.yml
 
+# The migrations are built each time
+tests/migrations/
```

Comment:
Have you tried passing the [`--nomigrations`](https://pytest-django.readthedocs.io/en/latest/database.html#nomigrations-disable-django-1-7-migrations) flag instead of creating the migrations on the fly solely for Django 2.2?

Reply:
I did try that.  I'll be very grateful if there is a better solution which works on Django 2.2, but I didnt find it.  As can be seen in logic and logs, I do avoid explicit migrations for other Django versions.  It is only needed for 2.2, and I can not reproduce the problem on openSUSE servers or my openSUSE workstation.

I havent tried to reproduce it by creating Debian images like Travis images.  That is more than I had time for.

Reply:
Honestly, I cannot think of anything that might cause that. I'll come back to it later once I find the time or try a similar approach on another project Django based project. If I find the culprit, I'll let you know.

All in all, thank you for your time and effort. This solution is fine by me and I'll gladly merge it. :slightly_smiling_face: